data/json/npcs/isolated_road/isolated_road_jay_convert.json

```diff
@@ -0,0 +1,1315 @@
+              "op": "==",
+              "value": 800
+            }
+          ]
+        },
+        "effect": [
+          { "u_add_var": "gunsmith_ammo_from", "type": "number", "context": "artisans", "value": "545" },
+          { "u_sell_item": "545", "count": 800 }
+        ],
+        "topic": "TALK_GUNSMITH_CONVERT_TO_TYPE"
+      },
+      {
+        "text": ".300 AAC",
+        "condition": {
+          "and": [
+            { "u_has_items": { "item": "300blk", "count": 200 } },
+            {
+              "u_compare_var": "gunsmith_ammo_ammount",
+              "type": "number",
+              "context": "artisans",
+              "op": "==",
+              "value": 200
+            }
+          ]
+        },
+        "effect": [
+          { "u_add_var": "gunsmith_ammo_from", "type": "number", "context": "artisans", "value": "300" },
+          { "u_sell_item": "300blk", "count": 200 }
+        ],
+        "topic": "TALK_GUNSMITH_CONVERT_TO_TYPE"
+      },
+      {
+        "text": ".300 AAC",
+        "condition": {
+          "and": [
+            { "u_has_items": { "item": "300blk", "count": 800 } },
+            {
+              "u_compare_var": "gunsmith_ammo_ammount",
+              "type": "number",
+              "context": "artisans",
+              "op": "==",
+              "value": 800
+            }
+          ]
+        },
+        "effect": [
+          { "u_add_var": "gunsmith_ammo_from", "type": "number", "context": "artisans", "value": "300" },
+          { "u_sell_item": "300blk", "count": 800 }
+        ],
+        "topic": "TALK_GUNSMITH_CONVERT_TO_TYPE"
+      },
+      { "text": "Forget it.", "topic": "TALK_GUNSMITH_SERVICES" }
+    ]
+  },
+  {
+    "id": "TALK_GUNSMITH_CONVERT_TO_TYPE",
+    "type": "talk_topic",
+    "dynamic_line": {
+      "u_compare_var": "gunsmith_ammo_from",
+      "type": "number",
+      "context": "artisans",
+      "op": "==",
+      "value": 556,
+      "yes": "Sure, 5.56 to what?",
+      "no": {
+        "u_compare_var": "gunsmith_ammo_from",
+        "type": "number",
+        "context": "artisans",
+        "op": "==",
+        "value": 76251,
+        "yes": "Sure, 7.62x51 to what?",
+        "no": {
+          "u_compare_var": "gunsmith_ammo_from",
+          "type": "number",
+          "context": "artisans",
+          "op": "==",
+          "value": 76239,
+          "yes": "Sure, 7.62x39 to what?",
+          "no": {
+            "u_compare_var": "gunsmith_ammo_from",
+            "type": "number",
+            "context": "artisans",
+            "op": "==",
+            "value": 545,
+            "yes": "Sure, 5.45x39 to what?",
+            "no": {
+              "u_compare_var": "gunsmith_ammo_from",
+              "type": "number",
+              "context": "artisans",
+              "op": "==",
+              "value": 300,
+              "yes": "Sure, 300 BLK to what?",
+              "no": "I don't understand you should have told me what rounds you want."
```

Comment:
```suggestion
              "no": "I don't understand, you need to tell me what rounds you want."
```

Reply:
This is a debug output if it shows up something is very broken.

I actually should have made that more clear probably